modbus:
  host: 0.0.0.0
  port: 9000
  holding_registers:
    parameters:
      target_pressure_pistons:
        address: 40001
        data_type: "uint16"
        value: 0
      target_pressure_material:
        address: 40002
        data_type: "uint16"
        value: 0
      system_state:
        address: 40003
        data_type: "uint16"
        value: 0
      pulse_train_high:
        address: 40004
        data_type: "uint16"
        value: 0
      pulse_train_medium:
        address: 40005
        data_type: "uint16"
        value: 0
      pulse_train_low:
        address: 40006
        data_type: "uint16"
        value: 0
      emergency_stop_state:
        address: 40007
        data_type: "uint16"
        value: 0
      solenoid_valves_state:
        address: 40008
        data_type: "uint16"
        value: 0
      alarm_state:
        address: 40009
        data_type: "uint16"
        value: 0
      alarm_pre_vacuum_limit_high:
        address: 40010
        data_type: "uint16"
        value: 0
      alarm_pre_vacuum_limit_low:
        address: 40011
        data_type: "uint16"
        value: 0
      alarm_pre_vacuum_distance:
        address: 40012
        data_type: "uint16"
        value: 0
    sensors:
      material_pressure:
        address: 40013
        data_type: "uint16"
        value: 0
      material_temperature:
        address: 40014
        data_type: "uint16"
        value: 0
      laser_distance:
        address: 40014
        data_type: "uint16"
        value: 0
